superset/db_engine_specs/netezza.py

```diff
@@ -31,7 +31,7 @@ class NetezzaEngineSpec(PostgresBaseEngineSpec):
         "P1D": "DATE_TRUNC('day', {col})",
         "P1W": "DATE_TRUNC('week', {col})",
         "P1M": "DATE_TRUNC('month', {col})",
-        "P3M": "DATE_TRUNC('quarter', {col})",
+        "P3M": "ADD_MONTHS(DATE_TRUNC('quarter', {col}), 1)",
```

Comment:
This is a breaking change, and I don't think a quarter plus 1 mouth is a correct solution.